webhook-service/handler/handler_test.go

```diff
@@ -698,6 +698,67 @@ func Test_HandleIncomingFinishedEventWithResultingError(t *testing.T) {
 
 }
 
+func Test_HandleIncomingFinishedEventWithJSON(t *testing.T) {
```

Comment:
is it possible to add a test case, where the string is not unmarshable, so the string is returned instead of JSON object ?

Reply:
I added this in the unit test, do you think it is needed in the component test too?